README.rst

```diff
@@ -130,6 +130,18 @@ You can inspect the version of openwisp-monitoring currently installed with::
 
     openwisp_monitoring --version
 
+Quality Assurance Checks
+------------------------
+
+We use `LuaFormatter <https://luarocks.org/modules/tammela/luaformatter>`_ to format lua files.
+Once it is installed, you can format all files by::
+
+    ./qa-format
+
+Alternatively, you can format specific file, e.g.::
+
+    lua-format -i --config=luaformat.config openwrt-openwisp-monitoring/files/sbin/netjson-monitoring.lua
```

Comment:
are you sure this second part is needed at all? The syntax may change in the future and we may forget to update this here, will people working on this module really need to know this?

Reply:
is still here though

Reply:
:grimacing: Remvoed again